android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -242,9 +254,26 @@ public void onIceGatheringChange(PeerConnection.IceGatheringState iceGatheringSt
         webRTCModule.sendEvent("peerConnectionIceGatheringChanged", params);
     }
 
+    public String getReactTagForStream(MediaStream mediaStream) {
+        String reactTag = null;
+        for (Iterator<Map.Entry<String, MediaStream>> i
+             = remoteMediaStreams.entrySet().iterator();
+             i.hasNext();) {
+            Map.Entry<String, MediaStream> e = i.next();
+            if (e.getValue().equals(mediaStream)) {
+                reactTag = e.getKey();
+                i.remove();
```

Comment:
Of course not, thanks ! It was a copy/paste mistake, but because it's used only in onRemoveStream the stream was supposed to be removed, but it should be done elsewhere.